lib/vagrant-libvirt/config.rb

```diff
@@ -545,6 +545,7 @@ def _handle_cdrom_storage(options = {})
         }.merge(options)
 
         cdrom = {
+          type: options[:type],
```

Comment:
It would seem like a default `type` of `raw` should be defined in the options hash above.